src/index.html

```diff
@@ -5,10 +5,47 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <nav>
+        <a href="index.html"><img src="images/logo.png" alt="logo"></a>
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a href="index.html" class="blue_link">Apple</a>
+            <div class="rectangle"></div>
```

Comment:
Instead of adding a separate div with class 'rectangle', use a pseudo-element (::before or ::after) in CSS to create the active link underline effect.

Reply:
https://github.com/serdiukm/layout_moyo-header.git I don't understand where to add the link